multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/model/ServiceOperation.java

```diff
@@ -1,18 +1,13 @@
 package org.cloudfoundry.multiapps.controller.core.model;
 
 import java.text.MessageFormat;
-import java.util.Map;
+import java.util.Objects;
 
-import org.apache.commons.collections4.MapUtils;
+import org.cloudfoundry.client.v2.serviceinstances.LastOperation;
 import org.cloudfoundry.multiapps.controller.core.Messages;
 
 public class ServiceOperation {
```

Comment:
This seems like it should be moved to our CF Java client fork.

Reply:
Done